DependencyInjection/Configuration.php

```diff
@@ -1,13 +1,9 @@
 <?php
-
```

Comment:
Hi @mgiraud! Thank you very much for this PR! I was just wondering, why did you remove all these blank lines?